edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/logs/ILogsUploader.cs

```diff
@@ -2,12 +2,15 @@
 namespace Microsoft.Azure.Devices.Edge.Agent.Core.Logs
 {
     using System;
+    using System.IO;
     using System.Threading.Tasks;
 
-    public interface ILogsUploader
+    public interface IRequestsUploader
```

Comment:
Why change this name?

Reply:
Change the file name too?

Reply:
Since it also uploads the support bundle, I made the name more generic. I will rename the file, missed that